src/utils/generateChartData.js

```diff
@@ -15,46 +15,129 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
 import { metricToChartName } from "../constants/metrics";
+import { METRICS_NOT_PROVIDED } from "../constants/errors";
+import logger from "./logger";
+
+export const NO_HUMANIZED_VALUE = (metric) =>
```

Comment:
I think these should probably not be SNAKE_CASED since they're really functions and not static, constant strings. `noHumanizedValue` and `noMetricData`?